sphinx_immaterial/autodoc_property_type.py

```diff
@@ -15,9 +15,9 @@
 
 
 def _get_property_return_type(obj: property) -> Optional[str]:
-    if obj.fget is None:
-        return None
-    doc = obj.fget.__doc__
+    if hasattr(obj, "fget") and obj.fget is not None:
```

Comment:
Could change this to just:

```python
fget = getattr(obj, "fget", None)
if fget is not None:
    obj = fget
```

since `hasattr` just calls `getattr` anyway.

Reply:
sure, this also doesn't raise the `mypy` error (even though the type-breaking reassignment is still there - just undetected this time).